repos/system_upgrade/common/libraries/fetch.py

```diff
@@ -57,7 +71,8 @@ def read_or_fetch(filename,
                   service=None,
                   allow_empty=False,
                   encoding='utf-8',
-                  data_stream=None):
+                  data_stream=None,
+                  allow_download=True):
```

Comment:
PR title says 'Ignore download by default' so this should be False, or?

Reply:
@MichalHe  good point. It depends. by default I am taking it as the behaviour when running `leapp (pre)upgrade`, however, from the function perspective, it made sense to me to keep true to protect original behaviour, but change the invocation of this function. From the perspective of our repositories, it's practically same, but it could have effect on 3rd-party content. wdyt in this case? Do we want to go here with the default False or True? I incline more to the second way due to this, but switching it to `False` as a default is also reasonable, from the POV that people should be aware of the change.

I will think about it yet, as this is small change, I will keep the decision about that as the last thing, for now, I will prepare the rest of the code and will see what we choose in the end. Anyway, nice catch during the review :-) I like that you think about it.

Reply:
@oamg/developers ^ headsup, in case you want to add your opinion :)

btw, it will be a short timer - this is going to be merged in couple of days (definitely this week).

Reply:
ah. just realized that part of that has been covered by https://github.com/oamg/leapp-repository/pull/1121 - which means the `allow_download=True` is already present publicly. so I would stay keeping it as this and change the invocation only.

Reply:
My apologize for the noise :pray: :brain: